README.md

```diff
@@ -1614,6 +1615,162 @@ authClient.tokenManager.off('renewed');
 authClient.tokenManager.off('renewed', myRenewedCallback);
 ```
 
+## Using the library in Node as an Authentication SDK
+
+> :warning: Beta alert! This library is in beta.
+
+You can use this library on server side in your Node application as an Authentication SDK. It can only be used in this way for communicating with the [Authentication API](https://developer.okta.com/docs/api/resources/authn), **not** to implement an OIDC flow.
+
+To include this library in your project, you can follow the instruction in the [Getting started](#getting-started) section.
+
+### Configuration
+
+You only need to set the `url` for your Okta Domain:
+
+```javascript
+var OktaAuth = require('@okta/okta-auth-js');
+
+var config = {
+  // The URL for your Okta organization
+  url: 'https://{yourOktaDomain}'
+};
+
+var authClient = new OktaAuth(config);
+```
+
+### API Reference
+
+Since the Node library can be used only for the Authentication flow, it implements only by a subset of okta-auth-js APIs:
+
+* [signIn](#signinoptions-in-node)
+* [forgotPassword](#forgotpasswordoptions-in-node)
+* [unlockAccount](#unlockaccountoptions-in-node)
+* [verifyRecoveryToken](#verifyrecoverytokenoptions-in-node)
+* [tx.resume](#txresume)
+* [tx.exists](#txexists)
+* [transaction.status](#transactionstatus)
+  * [LOCKED_OUT](#locked_out)
+  * [PASSWORD_EXPIRED](#password_expired)
+  * [PASSWORD_RESET](#password_reset)
+  * [PASSWORD_WARN](#password_warn)
+  * [RECOVERY](#recovery)
+  * [RECOVERY_CHALLENGE](#recovery_challenge)
+  * [MFA_ENROLL](#mfa_enroll)
+  * [MFA_ENROLL_ACTIVATE](#mfa_enroll_activate)
+  * [MFA_REQUIRED](#mfa_required)
+  * [MFA_CHALLENGE](#mfa_challenge)
+  * [SUCCESS](#success)
+
+### `signIn(options) in Node`
```

Comment:
Let's not duplicate examples, rather let's just use this "Node JS Usage" section to have a list of things that are/aren't possible with Node.  For example, list methods that can be used, mention ones that can't, such as set cookie and redirect.